### PR TITLE
pkg/kubelet/dockertools/docker_manager.go: removing unused stuff

### PR DESCRIPTION
**Pull request by php-coder**
This PR removes unused constants and variables. I checked that neither kubernetes nor openshift code aren't using them.

### CODE REVIEW

File: pkg/kubelet/dockertools/docker_manager.go [hunk 1]
```diff
@@ -338,12 +338,6 @@ func GetContainerLogs(client DockerInterface, pod *v1.Pod, containerID kubeconta
 }
 
 var (
-	// ErrNoContainersInPod is returned when there are no containers for a given pod
-	ErrNoContainersInPod = errors.New("NoContainersInPod")
-
-	// ErrNoPodInfraContainerInPod is returned when there is no pod infra container for a given pod
-	ErrNoPodInfraContainerInPod = errors.New("NoPodInfraContainerInPod")
```
Comment (mtaufen) [inline pkg/kubelet/dockertools/docker_manager.go]: @yujuhong ok to remove these error reasons, or were you planning on using them?
Seemed usage of `ErrNoContainersInPod` stopped in bb417e82d7e40bb94d1928a01e06b8d7b1e6fa00 -- `pkg/kubelet/dockertools/manager.go` and `ErrNoPodInfraContainerInPod` doesn't look like it was ever in use as far as I can tell (note I haven't searched back before the origin of `pkg/kubelet/dockertools/manager.go`).

**Review by mtaufen — CHANGES_REQUESTED** [review]

**Comment by mtaufen** [comment]
@php-coder If there is no release note, you can just omit that entire section from the initial comment.

File: pkg/kubelet/dockertools/docker_manager.go (hunk 1 above)
Comment (yujuhong) [inline pkg/kubelet/dockertools/docker_manager.go]: Don't think we are going to use them, especially now that we are transitioning to CRI. Most of the code in this package (especially those mentioning "infra container") will be deprecated eventually.

File: pkg/kubelet/dockertools/docker_manager.go [hunk 2]
```diff
@@ -196,18 +196,6 @@ type podGetter interface {
 	GetPodByUID(kubetypes.UID) (*v1.Pod, bool)
 }
 
-func PodInfraContainerEnv(env map[string]string) kubecontainer.Option {
```
Comment (php-coder) [inline pkg/kubelet/dockertools/docker_manager.go]: Thanks for the info. I've removed this commit.

**Review by mtaufen — APPROVED** [review]

**Comment by mtaufen** [comment]
/lgtm

Thanks for this work :)

**Pull request merged**